frontend/components/surplus/CollectSurplusAuctionPanel.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+    <BasePanel :current-state="currentStateAndTitle.name">
+        <template #title>
+            {{ currentStateAndTitle.title }}
+            <time-till v-if="auction.state === 'have-bids'" :date="auction.earliestEndDate" />
+        </template>
+        <TextBlock v-if="auction.state === 'ready-for-collection'">
+            Since the end of the auction at {{ auction.earliestEndDate.toUTCString() }}, the winner can now collect the
+            auction amount.
```

Comment:
Here I would adjust the text and also make a distinction between the case where the user == winner and user !== winner. 

For the case user == winner I propose: 
"Since the auction ended at {{ auction.earliestEndDate.toUTCString() }}, you can now collect the auction amount."

For the case user !== winner I propose:
"Since the auction ended at {{ auction.earliestEndDate.toUTCString() }}, you can now collect the auction amount. Note however that the auction amount will be transferred to the highest bidder's wallet."